### ISSUE TITLE
[Feature] Manipulation of Article Titles

### ISSUE THREAD

**Issue by nbgibson** [marienfressinaud/FreshRSS#4335]
**Is your feature request related to a problem? Please describe.**
A few of my feeds are provided by a service via my local library and add a [SERVICENAME] prefix to all articles provided by their service. This isn't a major issue but does become tiring over time and hinders readability to see the same block of text reproduced several dozen times each morning.

**Describe the solution you’d like**
The ability to add some basic string manipulation (find/replace, trim, ect.) on an on per subscription level would be greatly apricated. 

**Describe alternatives you’ve considered**
So far "ignoring", "dealing with", and "putting up with" the issue has provided workable results but may not be viable over long periods of time. I've also reached out to the provider regarding potentially altering their feeds, but since they are repackaging major newspapers I doubt they would be interested in voluntarily removing their stamp on the content. 

**Additional context**
This isn't load bearing or particularly critical by any means, but it would be a nice QoL feature for anyone having to deal with subscriptions that may not be formatted in a user friendly manner. My particular use case is just sourced in minor annoyance, but I could see some potential with this functionality in terms of accessibility.

**Comment by kupietools** [comment]
Agreed. I'd also like to be able to specify things like that the title should be displayed as "{{source name}} {{title}}" since source names disappear on an iPhone's narrow screen.

**Comment by Alkarex** [comment]
At the moment, it is already possible to make an extension to perform custom content manipulation.
But I agree that having core functions allowing regex manipulations could be interesting.

**Comment by aledeg** [comment]
I've started working on that.

**Comment by aledeg** [comment]
I've almost completed a first run. I will need some testers when it's available.
@nbgibson do you think you'll be able to test my work?

**Comment by nbgibson** [comment]
I don't see why not. Point me in the direction of how to wrangle getting that set up and I can poke at it.

**Comment by aledeg** [comment]
Great. I'll let you know when I am ready.

**Comment by aledeg** [comment]
@nbgibson the code and the documentation are ready. You can check the PR for more information.
To test that, you'll need to download the PR code and run it locally.
If you've never do that before, the current documentation is a good starting point to get the docker environment running.

**Comment by nbgibson** [comment]
Good stuff. I'll try and take a look at it over the weekend, baby allowing.
